must-gather/internal/metrics/collector.go

```
@@ -0,0 +1,109 @@
+	if err := resultPath.MkdirAll(); err != nil {
+		return err
+	}
+
+	// Execute curl command in Prometheus pod
+	cmd := []string{"/bin/bash", "-c",
+		fmt.Sprintf("curl -sG http://localhost:9090/api/v1/%s", object)}
+
+	output, err := m.client.ExecInPod(ctx, "openshift-monitoring", pod, "prometheus", cmd)
+	if err != nil {
+		// Write error to stderr file
+		resultPath.Add("error.log").WriteFile([]byte(err.Error()))
```

golangci-lint (2.12.2)

[error] Error return value of `(github.com/openshift/cluster-logging-operator/must-gather/internal/api.Path).WriteFile` is not checked

(errcheck)